packages/embeds/embed-core/src/preview.ts

```diff
@@ -8,40 +8,35 @@ const EMBED_LIB_URL = import.meta.env.NEXT_PUBLIC_EMBED_LIB_URL || `${WEBAPP_URL
 
 // Install Cal Embed Code Snippet
 (function (C, A, L) {
-  // @ts-ignore
-  let p = function (a, ar) {
+  const p = function (a: any, ar: any) {
```

Comment:
same as above regarding the use of `any`

Reply:
This is actually the embed snippet that we gave to users. This is already typed in the source. Didn't want to re-type it here.